fix: resolve 'cloud' property type error in app.config.ts

## Problem
TypeScript compilation error in `src/app.config.ts`:

```
src/app.config.ts(47,3): error TS2353: Object literal may only specify known properties, and 'cloud' does not exist in type 'AppConfig'.
```

## Solution
Either:
1. Extend the `AppConfig` type to include `cloud` property, or
2. Use type assertion if this is a valid WeChat miniprogram config option not yet typed by Taro

--- a/src/app.config.ts
+++ b/src/app.config.ts
@@ -44,5 +44,6 @@ export default defineAppConfig({
       },
     ],
   },
+  // @ts-expect-error WeChat cloud development - not typed in Taro's AppConfig
   cloud: true,
 });

